worker.py

```diff
@@ -890,6 +899,57 @@ def bag_full(self):
                     self.logger.warning("Failed to apply incubator {} on {}, code: {}".format(
                         inc.get('id', 0), egg.get('id', 0), ret))
 
+    async def handle_captcha(self, responses):
+        if not config.CAPTCHA_KEY:
+            self.logger.error("CAPTCHA Key is not set! this should not happen")
+            raise CaptchaException
+
+        if self.num_captchas >= config.CAPTCHA_RETRIES:
+            self.logger.error("User {} encountered a CAPTCHA to often, removing".format(self.username))
+            raise CaptchaException
+
+        self.num_captchas += 1
+
+        url = responses.get('CHECK_CHALLENGE', {}).get('challenge_url', ' ')
+        site_key = '6LeeTScTAAAAADqvhqVMhPpr_vB9D364Ia-1dSgK'
+        s = requests.Session()
```

Comment:
Ideally this would use [aiohttp](https://aiohttp.readthedocs.io/en/stable/) instead of requests so that it's not blocking, especially since this function is already a coroutine. I would also maybe make the session a class variable so that it can be shared and reused.

I can help if you're unsure of how to do that, or here's an example with my port of pgoapi from requests to aiohttp: Noctem/pgoapi@7660a8031d8979aa780bb04d9fd031ecb135da6c

The develop branch will switch to using that pgoapi branch soon, so aiohttp will be a dependency anyways.